Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -176,11 +176,16 @@ private void LateUpdate()
         {
```

Comment:
Since you're in here, can you update the `SolverHandler.Start()` method to respect the offset values? I found a bug earlier this week, but haven't had time to open a PR.

`        private void Start()`
        `{`
            `// TransformTarget overrides TrackedObjectToReference`
            `if (!transformTarget)`
            `{`
                `AttachToNewTrackedObject();`
            `}`
            `else`
            `{`
                `// we need to ensure the offset gets applied`
                `TrackTransform(transformTarget);`
            }``
        `}`

Reply:
It might make sense to do this in a separate PR (with it's own tests and focus).

This request is generally okay if you have significant test coverage and can be confident about the safety of incremental and tangential changes, but without it, you have to do another manual test pass (potentially holding up this change, etc)